src/Microsoft.DotNet.Compiler.Common/CultureInfoCache.cs

```diff
@@ -0,0 +1,439 @@
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.DotNet.Cli.Compiler.Common
+{
+    /// <summary>
+    /// Contains a list of known culture names that can be used to create a <see cref="System.Globalization.CultureInfo"/>.
+    /// </summary>
+    internal static class CultureInfoCache
+    {
+        /// <summary>
+        /// This list of known cultures was generated by CultureInfoGenerator using .NET Framework 4.6 RC or later on
+        /// Microsoft Windows NT 6.2.9200.0.
+        /// As new versions of .NET Framework and Windows are released, this list should be regenerated to ensure it
+        /// contains the latest culture names.
+        /// </summary>
+        public static readonly HashSet<string> KnownCultureNames = new HashSet<string>(StringComparer.OrdinalIgnoreCase)
```

Comment:
bummer it's not somewhere in the System.Text - should you file an issue on dotnet/corefx?